### PR TITLE
Add test for firetactoe.

### PR DESCRIPTION
**Pull request by jerjou**

### CODE REVIEW

File: appengine/standard/firebase/firetactoe/firetactoe_test.py [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+
+class MockHttp(object):
+    """Mock the Http object, so we can set what the response will be."""
+    def __init__(self, status, content=''):
+        self.content = content
+        self.status = status
+        self.request_url = None
+
+    def __call__(self, *args, **kwargs):
+        self.constructor_args = (args, kwargs)
```
Comment (jonparrott) [inline appengine/standard/firebase/firetactoe/firetactoe_test.py]: This doesn't seem to be used, remove it?

**Comment by jerjou** [comment]
@dpebot would you mind merging when travis approves?

**Comment by jerjou** [comment]
FYI added a commit to fix the style errors that are breaking the build.

**Comment by jerjou** [comment]
FYI removed a file that also had style errors (and a dubious null check). Also, a rewrite of the doc that uses it may make the file obsolete (so not worth trying to fix it). If it turns out to still be needed, we can open another PR to re-add it.

**Pull request merged**